tests/unit/test_bo.py

```diff
@@ -122,3 +123,113 @@ def test_acquisition():
     assert new.shape == (n2, n_params)
     assert np.all((new[:, 0] >= bounds['a'][0]) & (new[:, 0] <= bounds['a'][1]))
     assert np.all((new[:, 1] >= bounds['b'][0]) & (new[:, 1] <= bounds['b'][1]))
+
+
+class Test_MaxVar:
+    """Run a collection of tests for the MaxVar acquisition."""
+
+    def test_acq_bounds(self, acq_maxvar):
+        """Check if the acquisition is performed within the bounds.
+
+        Parameters
+        ----------
+        acq_maxvar : MaxVar
+            Acquisition method.
+
+        Notes
+        -----
+        - acq_maxvar acquires points from a 2-D Gaussian noise model.
+
+        """
+        bounds = acq_maxvar.model.bounds
+        batch_size = 2
```

Comment:
I guess we should allocate a bit more resources to make the test more reasonable. Would it take long with 100?

Reply:
It depends on the acquisition function, 100 MaxVar acquisitions would be fast but 100 RandMaxVar acquisitions would be slow. I set a small number to keep the complete testing time minimal.

Can increase the numbers for now to keep the testing time appropriate, and alter them upon addressing and defining the testing policy for stochastic algorithms.